installing_on_kubernetes/_topics/migrate_from_appliance.md

```diff
@@ -0,0 +1,131 @@
+## Preparing the Kubernetes namespace
+The deploy directory referenced below can be found in [the manageiq-pods repo](https://github.com/ManageIQ/manageiq-pods/tree/master/manageiq-operator/deploy)
+
+1. Search for the Custom Resource Definition (CRD) and create it if it doesn't already exist.
+```bash
```

Comment:
if you indent these blocks 3 spaces, they will render as part of number 1 instead of after it.  Not sure if we follow this pattern elsewhere in the docs.